piano-hero/core/src/main/java/com/novoda/pianohero/KeyListener.java

```diff
@@ -0,0 +1,8 @@
+package com.novoda.pianohero;
+
+interface KeyListener {
```

Comment:
commit b86cc27fc4eb0c73fd4903956dd801b1f4806b77
Author: Ataul Munim <hello@ataulm.com>
Date:   Tue Apr 18 09:21:08 2017 +0100

    Rename KeyListener to NoteListener